sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1748,182 +1748,6 @@ object functions {
    */
   def length(columnName: String): Column = length(Column(columnName))
 
-  //////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
note that this previously cut right into the middle of string functions so I moved them